exporting/prometheus/multi_metadata.yaml

```diff
@@ -0,0 +1,436 @@
+# yamllint disable rule:line-length
+---
+- &promexport
+  id: 'export-prometheus-remote'
+  meta: &meta
+    name: 'Prometheus Remote Write'
+    link: 'https://prometheus.io/docs/operating/integrations/#remote-endpoints-and-storage'
+    categories:
+      - export
+    icon_filename: 'prometheus.png'
+  keywords:
+    - exporter
+    - Prometheus
+    - remote write
+    - time series
+  overview:
+    exporter_description: |
+      Use the Prometheus remote write exporting connector to archive your Netdata metrics to the external storage provider of your choice for long-term storage and further analysis.
+    exporter_limitations: 'The remote write exporting connector does not support buffer on failures.'
+  setup:
+    prerequisites:
+      list:
+        - title: ''
+          description: |
+            - Netdata and the external storage provider of your choice, installed, configured and operational.
+            - `protobuf` and `snappy` libraries installed.
+            - Netdata reinstalled after the libraries.
+    configuration:
+      file:
+        name: 'exporting.conf'
+      options:
+        description: 'The following options can be defined for this exporter.'
+        folding:
+          title: 'Config options'
+          enabled: true
+        list:
+          - name: 'enabled'
+            default_value: 'no'
+            description: 'Enables or disables an exporting connector instance (yes|no).'
+            required: true
+          - name: 'destination'
+            default_value: 'no'
+            description: 'Accepts a space separated list of hostnames, IPs (IPv4 and IPv6) and ports to connect to. Netdata will use the first available to send the metrics.'
+            required: true
+            detailed_description: |
+              The format of each item in this list, is: [PROTOCOL:]IP[:PORT].
+              - PROTOCOL can be udp or tcp. tcp is the default and only supported by the current exporting engine.
+              - IP can be XX.XX.XX.XX (IPv4), or [XX:XX...XX:XX] (IPv6). For IPv6 you can to enclose the IP in [] to separate it from the port.
+              - PORT can be a number of a service name. If omitted, the default port for the exporting connector will be used.
+  
```

Comment:
Hello @shyamvalsan ,

This PR also have warnings from `yamllint`, please, take a look on them.

Best regards!